packages/astro-rss/src/index.ts

```diff
@@ -28,6 +28,8 @@ type RSSOptions = {
 	stylesheet?: string | boolean;
 	/** Specify custom data in opening of file */
 	customData?: string;
+	/** Whether to include drafts or not */
+	drafts?: boolean;
```

Comment:
nit: I'd prefer the name `includeDrafts` for naming a boolean option

Reply:
I like this but it would then be inconsistent with our own markdown config which uses `drafts: true` to build draft files as pages.

I guess we _could_ also rename the markdown flag in the future although `markdown.includeDrafts` is a less obvious home run in terms of naming.

Reply:
Fair enough! I personally didn't make the connection between the markdown config and the RSS config, so the different naming didn't bother me. I'm okay keeping it as `drafts` if it makes documentation easier.